src/main/resources/schema.sql

```diff
@@ -46,7 +46,10 @@ CREATE TABLE county (
     id BIGSERIAL PRIMARY KEY,
```

Comment:
The counties that are "special":

Brooks - Has 2 Polygons (MultiGeometry)
Lee - Has 2 Polygons (MultiGeometry)
Macon - Has 2 Polygons (MultiGeometry)
Pike - Has a hole (innerBoundaryIs)
Taylor - Has 3 Polygons (MultiGeometry)
Terrell - Has a hole (innerBoundaryIs)
Schley - Has a hole (innerBoundaryIs)
Upson - Has 2 Polygons (MultiGeometry)


Example insert of a "regular" county:

INSERT INTO county (
    name,
    state,
    outer_boundary1,
    inner_boundary1,
    outer_boundary2,
    outer_boundary3
) VALUES (
    'Appling',
    'GA',
    '-82.13584,31.8922 -82.12684,31.89 ...',
    NULL,
    NULL,
    NULL
);


Example insert of a county with a hole:

INSERT INTO county (
    name,
    state,
    outer_boundary1,
    inner_boundary1,
    outer_boundary2,
    outer_boundary3
) VALUES (
    'Pike',
    'GA',
    '-84.53702,33.03646 -84.5276,33.04416 ...',
    '-84.29866,32.99967 -84.29867,33.00414 ...',
    NULL,
    NULL
);


Example insert of a county with 2 polygons:

INSERT INTO county (
    name,
    state,
    outer_boundary1,
    inner_boundary1,
    outer_boundary2,
    outer_boundary3
) VALUES (
    'Brooks',
    'GA',
    '-83.31153,30.63457 -83.31187,30.63459 ...',
    NULL,
    '-83.73743,30.9872 -83.73729,30.99913 ...',
    NULL
);


Example insert of a county with 3 polygons:

INSERT INTO county (
    name,
    state,
    outer_boundary1,
    inner_boundary1,
    outer_boundary2,
    outer_boundary3
) VALUES (
    'Taylor',
    'GA',
    '-84.05152,32.52205 -84.05136,32.52205 ...',
    NULL,
    '-84.37282,32.39755 -84.37236,32.39755 ...',
    '-84.45156,32.54235 -84.4441,32.55485 ...'
);